Directory.Build.props

```diff
@@ -267,6 +267,11 @@
     <!-- Disable source link on local builds. -->
     <EnableSourceLink Condition="'$(ContinuousIntegrationBuild)' != 'true' and '$(OfficialBuildId)' == ''">false</EnableSourceLink>
   </PropertyGroup>
+  
+  <PropertyGroup>
+    <!-- Embed IBC data on Windows release builds, if IBCMerge isn't available this will just log the commandline -->
+    <EnablePartialNgenOptimization Condition="'$(EnablePartialNgenOptimization)' == '' and '$(OS)' == 'Windows_NT' and '$(ConfigurationGroup)' == 'Release'">true</EnablePartialNgenOptimization>
```

Comment:
>'$(OS)' == 'Windows_NT [](start = 91, length = 22)

Probably want to add `'$(DotNetBuildFromSource)' != 'true'` in case someone runs source build locally on Windows machine.

Reply:
Sure I can do this.